Loop/Managers/LoopDataManager.swift

```diff
@@ -798,7 +798,7 @@ extension LoopDataManager {
                                       maximumBasalRatePerHour: loopSettings.maximumBasalRatePerHour,
                                       maximumBolus: loopSettings.maximumBolus,
                                       suspendThreshold: loopSettings.suspendThreshold,
```

Comment:
This was discussed in https://github.com/LoopKit/Loop/issues/1536... But basically loopSettings doeesn't store the deviceToken so this will be nil. We may want to just remove that property altogether to avoid the confusion but I'd need to verify that is ok.